utils/gen_onnx_mlir.py

```diff
@@ -425,6 +425,29 @@
    auto resultType = mlir::UnrankedTensorType::get(to.getValue());
    build($_builder, $_state, resultType, input, to);
   }] >
+  ];'''),
+  ('Split',
+ '''   let builders = [
+  OpBuilder<(ins "Value":$input, "IntegerAttr":$axis, "ArrayAttr":$split), [{
+   Type elementType = input.getType().cast<ShapedType>().getElementType();
+   std::vector<mlir::Type> outputTypes;
+   if (split.size() == 0) {
+     outputTypes.emplace_back(UnrankedTensorType::get(elementType));
+   } else {
+     std::vector<int64_t> inputShape = input.getType().cast<MemRefType>().getShape();
+     int64_t splitAxis = axis.cast<IntegerAttr>().getSInt();
+     int64_t splitNum = (split.size() == 0) ? inputShape[splitAxis] : split.size();
```

Comment:
split.size() == 0 is checked at 434. Will this condition always be false here?

Reply:
You are right. I removed the check. Thank you.